database/factories/EmployeeFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Employee;
+use Illuminate\Support\Str;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Faker\Factory as Faker;
+
+class EmployeeFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    // protected $model = Employee::class;
+
+    public function definition()
+    {
+        $faker = Faker::create();
+
+         return [
+            
+            'employee_code' => $faker->numerify("E###"),
```

Comment:
soale iku wes extends Factory jd iso ngene ae
```suggestion
            'employee_code' => $this->faker->numerify("E###"),
```